### ISSUE TITLE
dune-release lint could catch missing descr

### ISSUE THREAD

**Issue by emillon** [samoht/dune-release#68]
Hi,

Thanks for `dune-release`, it's useful to release things with dune :)

I just noticed a frustrating point : if the `descr` file is missing, or if somehow the description can not be extracted from `README.md`, `dune-release` will fail when when doing `dune-release opam pkg`. It'd be nice if it could be caught before tagging the release, for example at `dune-release lint` time.

Thanks!

**Comment by mmottl** [comment]
In my case there is a `descr` file, but it is apparently not being read.  Depending on the release step, `dune-release` may complain about a missing description or synopsis then.  This is a pretty annoying problem, because it prevents me from easily releasing new packages.

**Comment by samoht** [comment]
@mmottl which version of dune-release are you using?

**Comment by mmottl** [comment]
@samoht Version 1.1.0.  Interestingly, I cannot replicate this problem anymore, not even with older `dune-release` versions.  Is it possible that this bug was fixed in a library that `dune-release` depends on?  A short while ago I noticed that `dune-release` was rebuilt after some underlying packages changed, but I hadn't checked whether the problem still exists.  Anyway, the problem has apparently disappeared.
